src/components/ExhibitionCard/index.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import styles from './styles.module.css';
+
+const ExhibitionCard = ({ id, title, shortDescription, imageId, imageUrl, status, galleryTitle, startAt, endAt }) => {
+  let imageSrc = `${process.env.PUBLIC_URL}/noImage.jpg`;
+  if (imageUrl) {
+    imageSrc = imageUrl;
+  } else if (imageId) {
+    imageSrc = `https://www.artic.edu/iiif/2/${imageId}/full/300,/0/default.jpg`;
+  }
+
+  return (
+    <div className={styles.container}>
+      <Link to="/exhibitions/:exhibitionsId" key={id}>
```

Comment:
This route doesn't exist yet, please remove it for now so that the users don't see a 404 page when the click on an exhibition card.